addons/sale_crm/tests/test_sale_crm_revenue.py

```diff
@@ -0,0 +1,47 @@
+from odoo.addons.crm.tests import common as crm_common
```

Comment:
can we import TestCrmCommon instead? :)
from odoo.addons.crm.tests.crm_common import TestCrmCommon